components/autofill/src/sync/credit_card/mod.rs

```diff
@@ -105,6 +106,10 @@ struct PayloadEntry {
     #[serde(rename = "timesUsed")]
     pub times_used: i64,
     pub version: u32, // always 3 for credit-cards
+    // Fields that the current schema did not expect, we store them only internally
+    // to round-trip them back to sync without processing them in any way
+    #[serde(flatten)]
+    pub unknown_fields: UnknownFields,
```

Comment:
but even then, you probably don't need this `pub` at all? At least, addresses doesn't have it.